candig/server/serverconfig.py

```diff
@@ -202,18 +202,6 @@ class TykConfig(KeycloakOidConfig):
     TYK_SERVER = 'http://ga4ghdev01.bcgsc.ca:8008'
```

Comment:
Maybe replace by http://localhost:8080 or http://candig:8080 ...

Reply:
I changed it to localhost